### PR TITLE
fix: rename dialog title

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes # mf-40

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Review by Jack-Works — DISMISSED** [review]

File: packages/mask/src/plugins/Wallet/SNSAdaptor/SelectTokenDialog.tsx [hunk 1]
```diff
@@ -101,7 +101,7 @@ export function SelectTokenDialog(props: SelectTokenDialogProps) {
             titleBarIconStyle={isDashboard ? 'close' : 'back'}
             open={open}
             onClose={onClose}
-            title={t('plugin_wallet_select_a_token')}>
+            title={isDashboard ? t('add_token') : t('plugin_wallet_select_a_token')}>
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Wallet/SNSAdaptor/SelectTokenDialog.tsx]: What?! Try to swap any token with this patch on the dashboard page.

**Comment by wenluomask** [comment]
Checked.

**Comment by guanbinrui** [comment]
> Checked.

Try to swap any token with this patch on the dashboard page.

**Comment by wenluomask** [comment]
Do you mean swap the token just added? It works.

**Review by hanszhoou — DISMISSED** [review]

**Comment by guanbinrui** [comment]
@lelenei @wenluomask Try to swap a token on the dashboard, and click the select token button. The title should be `Select a Token` rather than `Add a Token`.

image

**Comment by UncleBill** [comment]
> @lelenei @wenluomask Try to swap a token on the dashboard, and click the select token button. The title should be `Select a Token` rather than `Add a Token`.



If we have https://github.com/DimensionDev/Maskbook/pull/5632 merged, it will be simple

```ts
pickToken({
  title: 'Add a Token' // or 'Select a Token'
})
```

**Review by guanbinrui — APPROVED** [review]

**Comment by guanbinrui** [comment]
@DimensionDev/qa

**Comment by wenluomask** [comment]
Checked.

**Pull request merged**